LICENSE.txt

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Why do we choose GNU license?

Reply:
I don't see a difference between them so I chose this one quite randomly. It forbids to create closed repos from ours and I thought  it's not bad